template_files/object.h

```diff
@@ -0,0 +1,4 @@
+#ifndef {objectSpecCap}_H
+#define {objectSpecCap}_H 1
```

Comment:
```suggestion
#define {objectSpecCap}_H
```
not consistent with the other header, tbh I don't think setting a value is that necessary